nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -141,6 +141,7 @@ def __init__(
         overflow_fix: Optional[OverflowFix] = None,
         quantize_outputs: bool = False,
         inplace_statistics: bool = True,
+        statistics_per_sample: bool = False,
```

Comment:
Why this option wasn't introduced for all algorithms?

Reply:
Statistics calculation now is changed only for MinMax

Reply:
The reason for this PR was to introduce the batch support for the whole PTQ part, wasn't it?

Reply:
Yes, I think that there is a need to add a description regarding each algorithm status. I will add it